Src/AutoFixtureUnitTest/AbstractRecursionIssue/Repro.cs

```diff
@@ -65,7 +65,7 @@ public void Workaround()
                     typeof(ItemBase),
                     typeof(FunkyItem)));
 
-            Assert.DoesNotThrow(() => fixture.Create<FunkyItem>());
+            AssertEx.DoesNotThrow(() => fixture.Create<FunkyItem>());
```

Comment:
I've created this method because [DoesNotThrow](https://github.com/xunit/xunit/issues/188) method has been removed in xunit 2.0 and I wanted to quickly migrated all the code.

Do you want me to create a test for the `AssertEx` method? Also I could replace this with the following pair as advised [here](https://github.com/xunit/xunit/issues/188#issuecomment-86584802):
```
var ex = Record.Exception(() => sut.Foo());
Assert.Null(ex);
```